core/Cookie.php

```diff
@@ -227,9 +229,9 @@ private function extractSignedContent($content)
      * Unserialize the array when necessary.
      * Decode the non numeric values that were base64 encoded.
      */
-    protected function loadContentFromCookie()
+    protected function loadContentFromCookie($validateSignature = true)
```

Comment:
Not too important but maybe the parameter doesn't need to have a default value?

Reply:
As this function is protected it maybe could break classes that extend it (some plugins maybe)? I put the default value for safety, but if you do not like it, I have nothing against removing it :)